addons/mail/static/src/models/thread_cache/thread_cache.js

```diff
@@ -56,6 +59,9 @@ function factory(dependencies) {
                 extraDomain: [['id', '>', Math.max(...messageIds)]],
                 limit: false,
             });
+            if (!fetchedMessages) {
+                return;
+            }
```

Comment:
Isn't this part specifically a fix that should be done in 14? What does it fix to not add the hint if there's not any message loaded? A part from code clean up which is good in master for sure.

Reply:
I don't think it's fixing anything, it's just useless to add the hint if no messages have been fetched